packages/design-system-tokens/src/themes/core/defaultTheme.ts

```diff
@@ -287,6 +291,22 @@ export const components = {
   spinner__color: themeColors.base,
   'spinner__background-color--inverse': themeColors['background-inverse'],
   'spinner__color--inverse': themeColors.white,
+  // text input
+  'text-input__line-height': 1.3,
+  'text-input__background-color--disabled': themeColors['gray-lighter'],
+  'text-input__border-width': '2px',
+  'text-input__border-color': themeColors.base,
+  'text-input__border-color--disabled': themeColors['gray-light'],
+  'text-input__border-color--error': themeColors.error,
+  'text-input__border-color--error--inverse': themeColors['error-light'],
+  'text-input__border-color--inverse': themeColors.black,
+  'text-input__border-color--success': themeColors['success-light'],
+  'text-input__color': themeColors.base,
+  'text-input__max-width': '460px',
```

Comment:
@line47, do you imagine them all being a consistent max-width? With the theme settings, each of your variables is always allowed to have its own value, but the alias would just be tracking _what's the default max-width for a form element_?

Reply:
@pwolfert a token like this could be useful for setting a base default max-width for forms with the flexibility to override. I also think more exploration is needed.

Reply:
I've made these changes to move the max-width to `form__max-width` and have updated the associations from the components that were set individually. So working as it always has, but definable from a single point now.